src/steps/powershell.rs

```diff
@@ -78,17 +79,44 @@ impl Powershell {
     }
 
     pub fn update_modules(&self, ctx: &ExecutionContext) -> Result<()> {
-        print_separator(t!("Powershell Modules Update"));
-        let mut cmd_args = vec!["Update-Module"];
+        print_separator("Powershell Modules Update");
 
+        let mut cmd_args = vec!["-Command", "Update-Module"];
+
+        // Add -Force option if --yes is enabled
+        if ctx.config().yes(Step::Powershell) {
+            cmd_args.push("-Force");
+        }
+
+        // Add -Verbose option if verbose mode is enabled
         if ctx.config().verbose() {
             cmd_args.push("-Verbose");
         }
-        if ctx.config().yes(Step::Powershell) {
-            cmd_args.push("-Force");
+
+        #[cfg(not(windows))]
+        {
+            // Update user-scope modules WITHOUT sudo (to avoid root ownership issues)
+            println!("Updating user-scope modules...");
```

Comment:
This and `println!("Updating system-scope modules...");` are user-facing messages, which need translations